src/Parser/DeathKnight/Unholy/Modules/Features/VirulentPlagueUptime.js

```diff
@@ -14,24 +14,38 @@ class VirulentPlagueUptime extends Analyzer {
     enemies: Enemies,
   };
 
+  get vpUptime() {
```

Comment:
Since your class is already called `VirulentPlague`, it can be omitted from all properties.